Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/PredictorCorrector.java

```diff
@@ -195,8 +195,10 @@ protected boolean trigger(DiVector candidate, int gap, int baseDimension, DiVect
                 for (int i = dimensions - difference; i < dimensions; i++) {
                     remainder += candidate.getHighLowSum(i);
                 }
-                return thresholder.getAnomalyGrade(remainder * dimensions / difference, previousIsPotentialAnomaly,
-                        triggerFactor) > 0;
+                // simplifying the following since remainder * dimensions/difference corresponds
+                // to the
+                // impact of the new data since the last anomaly
+                return thresholder.getAnomalyGrade(remainder * dimensions / difference, previousIsPotentialAnomaly) > 0;
```

Comment:
Tested the change on AD. The precision does not change much after the change. Our benchmark does not have such small gap anomaly.

Reply:
Thanks. We now have a small gap test in the RCF library, so it can get tracked.